ui/src/app.js

```diff
@@ -1206,10 +1206,18 @@ function fiatComplete (tx) {
   $('.fiat_complete_state .fiat .js-amount').text(tx.fiat)
   $('.fiat_complete_state .sent-coins .crypto-address').text(tx.toAddress)
 
+  toggleBatchCounter('show')
+
   setState('fiat_complete')
 }
 
+function toggleBatchCounter(state) {
+  const opacity = state === 'show' ? 100 : 0
+  $('.batch').css('opacity', opacity)
+}
+
 function dispenseBatch (data) {
+  if (data.of === 1) toggleBatchCounter('hide')
```

Comment:
I meant that you should handle both cases: hide if ``data.of === 1`` and show if ``data.of > 1``.